genre.rb

```diff
@@ -0,0 +1,14 @@
+class Genre
+  attr_accessor :name, :items
+  attr_reader :id
+
+  def initialize(name)
+    @id = Random.rand(1..1000)
+    @name = name
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item
+  end
```

Comment:
When an item is added to a Genre object the Genre object itself should be associated with the genre property of the Item object. The Item object allows for associations to be made by calling its `add_genre` method with the Genre object as a parameter: e.g. `item.add_genre(self)`. Kindly add that association after pushing the Item into the items array of the Genre.